src/components/ApiAction/QueryList.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <b-row class="h-100 text-center mx-0" align-v="center" align-h="center">
+    <b-col cols="12" class="h-100 px-0">
+      <b-card class="h-100 backgroundCard" no-body>
+        <b-card-body class="d-flex flex-column text-center m-0 p-0 h-100">
+          <b-card-text class="px-1 py-3 h-100">
+            <b-row
+              align-v="center"
+              class="h-100"
+              v-if="!paginatedQueries.length"
+            >
+              <b-col>
+                <b-card title="No API actions saved...">
+                  <b-card-text>
+                    <p>You can save your API ctions to fill this menu...</p>
```

Comment:
```suggestion
                    <p>Your saved queries will appear in this list.</p>
```

Reply:
We now call that `API Actions`, I probably should rename everything in this PR...